owncloudComLibrary/src/test/java/com/owncloud/android/lib/resources/webfinger/responses/WebfingerResponseTest.kt

```diff
@@ -0,0 +1,50 @@
+package com.owncloud.android.lib.resources.webfinger.responses
+
+import com.squareup.moshi.JsonAdapter
+import com.squareup.moshi.JsonDataException
+import com.squareup.moshi.Moshi
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import java.io.File
+
+class WebfingerResponseTest {
+    lateinit var adapter: JsonAdapter<WebfingerJrdResponse>
+
+    private fun loadResponses(fileName: String) =
+        adapter.fromJson(File(fileName).readText())
+
+    @Before
+    fun prepare() {
+        val moshi = Moshi.Builder().build()
+        adapter = moshi.adapter(WebfingerJrdResponse::class.java)
+    }
+
+    @Test
+    fun `check rel in to much information - ok - correct rell is returned`() {
```

Comment:
I don't understand very well the name of this test, what does `rel` or `rell` refer to? Is it `to` or `too`? I would change its name, don't you think?
Also, when the test is for a correct case (`ok`, like in this case), do we put additional information like `correct rell is returned` if it is evident?